iaaip/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template "><head>
+  <meta charset="utf-8">
+  <title>
+    Start your appeal - Appeal an immigration or asylum decision - HMCTS
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+  <meta name="theme-color" content="#0b0c0c">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+
+  <link rel="shortcut icon" href="/assets/images/favicon.ico" type="image/x-icon">
+
+
+
+  <base href="https://localhost:3000">
+  <link href="/main.css" rel="stylesheet" type="text/css">
```

Comment:
why aren't you using the govuk-frontend css file?
https://github.com/hmcts/azure-shutter-pages/blob/master/default/govuk-frontend-ie8-3.4.0.min.css

https://github.com/hmcts/azure-shutter-pages/blob/master/default/index.html

Was this based off the default shutter page or the existing one?